FrontBundle/EventSubscriber/CheckRoutingCacheViabilitySubscriber.php

```diff
@@ -39,12 +41,25 @@ public function checkCacheFileAndRefresh(GetResponseEvent $event)
             return;
         }
 
-        $cacheDir = $this->router->getOption('cache_dir');
+        $router = $this->router;
+        if ($this->router instanceof ChainRouter) {
+            foreach ($this->router->all() as $router) {
+                if ($router instanceof OpenOrchestraRouter) {
+                    break;
```

Comment:
the variable assignment is done in the foreach, this way i will get the open orchestra router